sql/core/src/main/scala/org/apache/spark/sql/execution/command/createDataSourceTables.scala

```diff
@@ -62,6 +62,7 @@ case class CreateDataSourceTableCommand(table: CatalogTable, ignoreIfExists: Boo
         sparkSession = sparkSession,
         userSpecifiedSchema = if (table.schema.isEmpty) None else Some(table.schema),
         className = table.provider.get,
+        paths = table.storage.locationUri.toSeq,
         bucketSpec = table.bucketSpec,
         options = table.storage.properties).resolveRelation()
```

Comment:
Here, the options do not have `path`, right? If so, the external data source connector will not be able to get the value of `path`. This might break the existing connectors.